tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -292,6 +294,15 @@ class ExecutionTest {
   TEST_METHOD(SaturateTest);
   TEST_METHOD(SignTest);
   TEST_METHOD(Int64Test);
+  TEST_METHOD(SERBasicTest);
```

Comment:
Wondering how it was decided to insert these test methods here, rather than after the other test methods?

Reply:
Moved all SER test methods to the end of the list